accname/name/comp_hidden_not_referenced.html

```diff
@@ -13,16 +14,119 @@
 
 <p>Tests the <a href="https://w3c.github.io/accname/#comp_hidden_not_referenced">#comp_hidden_not_referenced</a> portions of the AccName <em>Name Computation</em> algorithm.</p>
 
-<h2 class="ex" data-expectedlabel="heading label" data-testname="heading with interior hidden node">
-  heading
-  <span hidden>bogus</span>
-  label
+<button
+  class="ex"
+  data-expectedlabel="visible to all users"
+  data-testname="#1: button containing unreferenced text nodes that are hidden, visible to all users, and hidden from all users"
```

Comment:
Let's remove `#1: `, `#2: ` etc from the test names, and we might decide to insert some in the middle, and then our choice is to break the history, or have a misordered list, or insert something silly like `#1.5: `